### DISCUSSION TITLE
Couldn't update GFWLists

### DISCUSSION THREAD

**Discussion by dxyzx0** [v2rayA/v2rayA#1535] [Q&A]
If I press update it gives the following error:
```
Get "https://raw.githubusercontent.com/v2rayA/dist-v2ray-rules-dat/202410082212/geosite.dat": dial tcp 0.0.0.0:443: connect: connection refused
```

I found that this file seems not existing. Is it a bug or do I miss something? 
Thanks

**Comment by FinleyChan** [comment]
I couldn't update either, so I replaced the geo file manually

**Reply by stan1233** [reply]
Excuse me, how to replace the geo file manually?

**Comment by gsssz** [comment]
find a useful server in other client then update GFWlist
